diff --git i/llvm/lib/Support/Triple.cpp w/llvm/lib/Support/Triple.cpp
index 4f483c965..09b95f4c6 100644
--- i/llvm/lib/Support/Triple.cpp
+++ w/llvm/lib/Support/Triple.cpp
@@ -490,6 +490,8 @@ static Triple::VendorType parseVendor(StringRef VendorName) {
     .Case("mesa", Triple::Mesa)
     .Case("suse", Triple::SUSE)
     .Case("oe", Triple::OpenEmbedded)
+    .Case("poky", Triple::OpenEmbedded)
+    .Case("wrs", Triple::OpenEmbedded)
     .Default(Triple::UnknownVendor);
 }
 
